ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,248 @@
+<script>
+  import { getContext } from "svelte";
+  import {
+    TransferState,
+    fromStore,
+    toStore,
+    amountStore,
+  } from "../src/transfer.ts";
+
+  import { Dropdown, ModalLayout, Rad } from "../DesignSystem/Component";
+  import Row from "../DesignSystem/Component/Transaction/Row.svelte";
+  import {
+    Avatar,
+    Button,
+    Icon,
+    Input,
+    Text,
+    Title,
+  } from "../DesignSystem/Primitive";
+
+  const { identity } = getContext("session");
+  const { orgs } = getContext("session");
+
+  const dropdownOptions = [];
+  dropdownOptions.push({
+    variant: "avatar",
+    value: identity.accountId,
+    avatarProps: {
+      variant: "circle",
+      title: identity.metadata.handle,
+      avatarFallback: identity.avatarFallback,
+      imageUrl: identity.imageUrl,
+    },
+  });
+  for (let i = 0; i < orgs.length; i++) {
+    dropdownOptions.push({
+      variant: "avatar",
+      value: orgs[i].accountId,
+      avatarProps: {
+        variant: "square",
+        title: orgs[i].id,
+        avatarFallback: orgs[i].avatarFallback,
+        imageUrl: null,
+      },
+    });
+  }
```

Comment:
```suggestion
const dropdownOptions = [
    {
      variant: "avatar",
      value: identity.accountId,
      avatarProps: {
        variant: "circle",
        title: identity.metadata.handle,
        avatarFallback: identity.avatarFallback,
        imageUrl: identity.imageUrl,
      },
    },
    ...orgs.map(org => ({
      variant: "avatar",
      value: org.accountId,
      avatarProps: {
        variant: "square",
        title: org.id,
        avatarFallback: org.avatarFallback,
        imageUrl: null,
      },
    })),
  ];
```

Reply:
We're doing this in a few different places now; it might make sense to create a `payers`/`wallets` function in `transaction.ts` to get a list of all possible wallets. That might be beyond the scope of this PR but I think it'd be a lot easier than copying and pasting this every time the user wants to pay for something.

It could even be a `proxy` query for available wallets, and then `transaction.ts` just formats for display.

Reply:
That's a nice suggestion indeed. I propose we create an issue for that that we follow up in the next iteration since this whole wallets is already dragging as is as we have other concrete problems to solve first.

Reply:
nice one. Could one of you open an issue for this since this is outside this PR? Here's your suggestion, nice way of writing that :) https://github.com/radicle-dev/radicle-upstream/pull/712/commits/02a5bf1b159b69b0ab5d0056c2d251a93c40c6fe

Reply:
new issue: #728